### ISSUE TITLE
a11y: Dropdown menus are keyboard traps

### ISSUE THREAD

**Issue by roseleaf** [zendesk/volunteer_portal#12]
Keyboard users are currently forced to remain within a dropdown until they select an option.

Given that mouse users can click away without selecting an option, I think that this keyboard trap should be removed.

https://github.com/zendesk/volunteer/issues/424
